e2e/tests/selection/multi-selection-in-groups/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Runs in the **BROWSER**
+ * Imports are defined in 'e2e/imports.ts'
+ */
+
+import { Rect, Circle, Group, ClipPathLayout } from 'fabric';
+import { beforeAll } from '../../test';
+
+beforeAll(async (canvas) => {
+  canvas.setDimensions({ width: 700, height: 700 });
```

Comment:
I think you should reduce the size so the test has less whitespace

Reply:
i think it covers most of the surface, but yes i can shrink to the exact needed

Reply:
reduced